src/perf/lib/PerfCommon.h

```diff
@@ -24,6 +24,7 @@
 #define TPUT_DEFAULT_IDLE_TIMEOUT           (1 * 1000)
 
 #define RPS_MAX_CLIENT_PORT_COUNT           256
+#define RPS_MAX_REQUESTS_PER_SECOND         1500000 // 1.5 million RPS max as a guess
```

Comment:
I don't like depending on things like this. What happens when this ends up being not enough?

Reply:
It will only record up to this much. Any requests more then this just won't have latency numbers. There is logic to make sure we don't end up writing out of range.